### PR TITLE
DELETE SOON leo libraries versions update

### PR DESCRIPTION
**Pull request by aweng98**
\<your comments for this PR go here\>

Have you read [CONTRIBUTING.md](../CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've documented my API changes in Swagger

In all cases:

- [ ] Get two thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; you can delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

**Comment by aweng98** [comment]
jenkins retest

**Comment by aweng98** [comment]
jenkins retest

**Comment by aweng98** [comment]
jenkins retest

File: project/Settings.scala [hunk 1]
```diff
@@ -40,7 +40,7 @@ object Settings {
   val commonSettings =
     commonBuildSettings ++ commonAssemblySettings ++ commonTestSettings ++ List(
     organization  := "org.broadinstitute.dsde.workbench",
-    scalaVersion  := "2.12.3",
+    scalaVersion  := "2.12.6",
```
Comment (rtitle) [inline project/Settings.scala]: yay!

File: project/Version.scala [hunk 2]
```diff
@@ -1,17 +1,16 @@
 import sbt.Keys._
 import sbt._
 
-
 object Version {
   val baseVersion = "0.1"
 
   def getVersionString = {
-    def getLastCommitFromGit = { s"""git rev-parse --short HEAD""" !! }
+    //def getLastCommitFromGit = s"""git rev-parse --short HEAD""" !!
```
Comment (rtitle) [inline project/Version.scala]: Remove the commented method if this is going to be done inline.

File: project/plugins.sbt [hunk 3]
```diff
@@ -1,10 +1,10 @@
...
 
-addSbtPlugin("io.spray" % "sbt-revolver" % "0.8.0")
+addSbtPlugin("io.spray" % "sbt-revolver" % "0.9.1")
 
-addSbtPlugin("org.scoverage" % "sbt-coveralls" % "1.1.0")
+addSbtPlugin("org.scoverage" % "sbt-coveralls" % "1.2.5")
 
 // sbt-scoverage 1.5.0 is the first that supports scala 2.12.
-addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.5.0")
+addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.5.1")
```
Comment (rtitle) [inline project/plugins.sbt]: Huh, I didn't realize we have the scoverage plugins enabled. I don't think we're using the tool.

File: project/Dependencies.scala [hunk 4]
```diff
@@ -1,18 +1,18 @@
...
-  val workbenchUtilV    = "0.2-d97f551"
+  val akkaV         = "2.5.12"
+  val akkaHttpV     = "10.1.1"
+  val jacksonV      = "2.9.5"
+  val googleV       = "1.23.0"
+  val scalaLoggingV = "3.9.0"
+  val scalaTestV    = "3.0.5"
+  val slickV        = "3.2.3"
+
+  val workbenchUtilV    = "0.3-0e9d080"
```
Comment (rtitle) [inline project/Dependencies.scala]: I assume these wb-libs versions are the latest based on the README?

File: project/Dependencies.scala [hunk 5]
```diff
@@ -62,7 +62,7 @@ object Dependencies {
 
   val slick: ModuleID =     "com.typesafe.slick" %% "slick"                 % slickV
   val hikariCP: ModuleID =  "com.typesafe.slick" %% "slick-hikaricp"        % slickV
-  val mysql: ModuleID =     "mysql"               % "mysql-connector-java"  % "6.0.6"
+  val mysql: ModuleID =     "mysql"               % "mysql-connector-java"  % "8.0.11"
```
Comment (rtitle) [inline project/Dependencies.scala]: This is quite a big jump - should be compatible though I think.

**Comment by rtitle** [comment]
This is great!! A few comments above. Thanks for doing this

**Comment by rtitle** [comment]
Btw, I merged https://github.com/DataBiosphere/leonardo/pull/371 so if you rebase this branch, the build should pass.

File: jupyter-docker/Dockerfile [hunk 6]
```diff
@@ -129,6 +129,7 @@ RUN apt-get install -yq --no-install-recommends \
  && pip install google-cloud \
  && pip install --ignore-installed firecloud==0.16.7 \
  && pip install -U scikit-learn \
+ && pip install statsmodels==0.8.0 \
```
Comment (rtitle) [inline jupyter-docker/Dockerfile]: Weird that this is showing up in the diff, since it's already in dev. Do you need to rebase this branch?

**Comment by aweng98** [comment]
jenkins retest

**Pull request closed without merging**